### DISCUSSION TITLE
Not able to create Fulfillment Order when SKU has double quotes in it

### DISCUSSION THREAD

**Discussion by pawan1793** [amzn/selling-partner-api-models#3087] [General]
SKU with double quote in it is failing 
eg   sellerSKU  : **PH 7" HVL-1**  

Getting response as invalid SKU ...  as SKU is getting escaped when creating json object ,
when creating JSON object sku becomes **PH 7\" HVL-1**

**Comment by cccccroge** [comment]
+1
Mine is like `mag protector 10.2\"`, and will get the following response when create fulfillment order:
```.json
{
  "errors": [
    {
      "code": "InvalidInput",
      "message": "the SellerSKU for Item Id: 46743790 is invalid.",
      "details": ""
    }
  ]
}
```

Please fix this as the system doesn't treat double quote as invalid format. It should work for SP-API, too.

**Comment by syedhsan** [comment]
@cccccroge 
hey bro I am unable to get the SKU of items while getting the items details using SP API could you help please

**Reply by cccccroge** [reply]
1. What's the error message?
2. Do you have double quote in SKU of the item?
